packages/design-picker/src/components/style.scss

```diff
@@ -406,14 +406,46 @@
 .design-picker.design-picker__unified {
 	display: block;
 	padding-bottom: 200px;
+
 	.responsive-toolbar-group__dropdown .responsive-toolbar-group__grouped-list {
 		justify-content: flex-start;
 	}
+
 	.design-picker__grid {
 		@supports ( display: grid ) {
+			row-gap: 0;
```

Comment:
On the mobile beakpoint, we still need `row-gap: 32px`
|Before|After|
|------|-----|
|<img width="394" alt="Screen Shot 2565-10-28 at 16 08 24" src="https://user-images.githubusercontent.com/1881481/198550131-78f21074-d836-4f07-b184-2ad99d9770bf.png">|<img width="394" alt="Screen Shot 2565-10-28 at 16 08 04" src="https://user-images.githubusercontent.com/1881481/198550116-4ee29441-b178-4c96-8b6d-b03b606f103e.png">|

Reply:
Fixed! Unfortunately, setting `row-gap` means that it will still be applied to the hidden generated designs even though they are set to `max-height: 0` 🥲